generator/src/lib.rs

```diff
@@ -3406,26 +3406,28 @@ pub fn write_source_code<P: AsRef<Path>>(vk_headers_dir: &Path, src_dir: P) {
 
     let feature_code = quote! {
         use core::ffi::*;
-        use crate::vk::bitflags::*;
-        use crate::vk::definitions::*;
-        use crate::vk::enums::*;
+        use super::bitflags::*;
+        use super::definitions::*;
+        use super::enums::*;
```

Comment:
What's the motivation for this change? Outside of inline modules, I usually prefer to avoid `super` since it makes it more difficult to move code around.

Reply:
I don't particularly plan to move this code around very often, and favor locality/shortest-path. When reading `crate::vk::` it confuses me, because I _thought_ I was already somewhere inside the `vk` yet going into the module hierarchy from the root causes a disconnect.

It also feels like it helps identify the remaining cases where things, i.e. specifically the types and helpers from `prelude.rs` are defined in the wrong spot, if they're referred across our top-level modules. A good example is `fn debug_flags()` that seems more fitting inside the `vk` module.

Reply:
Works for me.